README.markdown

```diff
@@ -296,15 +274,6 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
```

Comment:
Just found this gem in CVL. Looks like an unsolved crossword puzzle! Which is how this style looks to me 100% of the time.

![screen shot 2015-06-02 at 8 33 08 pm](https://cloud.githubusercontent.com/assets/246403/7952182/a00a34a2-0966-11e5-8961-ed6fb166f503.png)